apps/graphql/src/dataloaders/itinerariesHelpers.js

```diff
@@ -0,0 +1,69 @@
+// @flow
+
+import type {
+  ApiRouteItemType,
+  VehicleTypeType,
+} from './ItinerariesloaderTypes';
+
+export const getVehicle = (type: ?VehicleTypeType, uniqueNo: string) => ({
+  type: type ?? null,
+  uniqueNo: uniqueNo ?? null,
+});
+
+export const getTransporter = (transporter: ?string) => ({
+  name: transporter ?? null,
+});
+
+export const getItineraryType = (routes: ?Array<Array<string>>) => {
+  if (routes == null) {
+    return null;
+  }
+  if (routes.length === 1) {
+    return 'oneway';
+  }
+  if (routes.length === 2) {
+    return 'return';
+  }
+  return null;
+};
+
+export const getSectors = (
```

Comment:
Also I fail to understand what this function does.

And why do we have to call map on the function afterwards?
`getSectors(itinerary.route, itinerary.routes).map`

Reply:
This function should get list of the flights and some map which defines how the flight should be divided into sectors (see what is sector here: https://kiwi.wiki/kiwi/handbook/#/comms/glossary?id=itinerary-parts-examples):

We need it for example to calculate duration in your final destination:

<img width="339" alt="screenshot 2019-01-11 at 16 10 34" src="https://user-images.githubusercontent.com/858321/51832464-0e137980-22f5-11e9-805d-0fea44d8400d.png">


Example:

**List of the flights:**
```js
[
  {
    flyFrom: 'OSL',
    flyTo: 'BCN',
    ...
  },
  {
    flyFrom: 'BCN',
    flyTo: 'PRG',
    ...
  },
  {
    flyFrom: 'PRG',
    flyTo: 'OSL',
    ...
  },
]
```

**Map:**
```
[
  ['OSL', 'PRG'], 
  ['PRG', 'OSL']
]
```

**Output from function:**

```js
[
  [
    {
      flyFrom: 'OSL',
      flyTo: 'BCN',
      ...
    },
    {
      flyFrom: 'BCN',
      flyTo: 'PRG',
      ...
    },
  ],
  [
    {
      flyFrom: 'PRG',
      flyTo: 'OSL',
      ...
    },
  ],
]
```